rcl/test/CMakeLists.txt

```diff
@@ -42,12 +42,7 @@ endif()
 # prevents finding it repeatedly in each local scope
 ament_find_gtest()
 
-macro(test_target)
-  find_package(${rmw_implementation} REQUIRED)
```

Comment:
this still needs to be called to make sure that CMake can find the corresponding rmw implementation package before it calls `dlopen` in the test executables?

Reply:
dlopen does not relay on CMake to do anything here as long as the LD_LIBRARY_PATH is set up properly: https://linux.die.net/man/3/dlopen . AFAIK the only reason this find_package call is here, is that it is required to make ament_target_dependencies work.